lib/generators/templates/app.js.es6

```diff
@@ -0,0 +1,19 @@
+//= require ember-rails/application
+//
```

Comment:
should we move the app into a specific folder and then just require the whole tree?